native/src/main/java/io/ballerina/observe/trace/zipkin/sampler/RateLimitingSampler.java

```diff
@@ -4,42 +4,52 @@
  */
 package io.ballerina.observe.trace.zipkin.sampler;
 
+import io.opentelemetry.api.common.AttributeKey;
 import io.opentelemetry.api.common.Attributes;
 import io.opentelemetry.api.trace.SpanKind;
 import io.opentelemetry.context.Context;
-import io.opentelemetry.sdk.internal.SystemClock;
+import io.opentelemetry.sdk.common.Clock;
+import io.opentelemetry.sdk.internal.RateLimiter;
 import io.opentelemetry.sdk.trace.data.LinkData;
 import io.opentelemetry.sdk.trace.samplers.Sampler;
 import io.opentelemetry.sdk.trace.samplers.SamplingDecision;
 import io.opentelemetry.sdk.trace.samplers.SamplingResult;
 
+import java.text.DecimalFormat;
+import java.text.DecimalFormatSymbols;
 import java.util.List;
 
+import static io.opentelemetry.api.common.AttributeKey.doubleKey;
+import static io.opentelemetry.api.common.AttributeKey.stringKey;
+
 /**
- * This class is copied from https://github.com/open-telemetry/opentelemetry-java/blob/v1.0.0/sdk-extensions/
+ * This class is copied from https://github.com/open-telemetry/opentelemetry-java/blob/v1.32.0/sdk-extensions/
  * jaeger-remote-sampler/src/main/java/io/opentelemetry/sdk/extension/trace/jaeger/sampler/RateLimitingSampler.java.
  * This sampler uses a leaky bucket rate limiter to ensure that traces are sampled with a certain constant rate.
  */
 public class RateLimitingSampler implements Sampler {
     public static final String TYPE = "ratelimiting";
+    static final AttributeKey<String> SAMPLER_TYPE = stringKey("sampler.type");
```

Comment:
Any specific reason to use default visibility? 
Other constants are public or private?

Reply:
No specific reason. I changed them to be private.